core/meterer/meterer.go

```diff
@@ -0,0 +1,64 @@
+package meterer
+
+import (
+	"time"
+
+	"github.com/Layr-Labs/eigensdk-go/logging"
+)
+
+type TimeoutConfig struct {
+	ChainReadTimeout    time.Duration
+	ChainWriteTimeout   time.Duration
+	ChainStateTimeout   time.Duration
+	TxnBroadcastTimeout time.Duration
+}
+
+// network parameters (this should be published on-chain and read through contracts)
+type Config struct {
```

Comment:
It would be useful to have comments describing what these parameters mean.

Reply:
sounds good! added some comments